src/plugins/intel_cpu/src/utils/shape_inference/shape_inference.cpp

```diff
@@ -579,7 +579,6 @@ const IShapeInferCommonFactory::TRegistry IShapeInferCommonFactory::registry{
     _OV_OP_SHAPE_INFER_REG(ScatterElementsUpdate, entryIOC),
     _OV_OP_SHAPE_INFER_REG(ScatterNDUpdate, entryIO),
     _OV_OP_SHAPE_INFER_REG(Select, entryIO),
-    _OV_OP_SHAPE_INFER_REG(Select, entryIO),
```

Comment:
Is it intentional?

Reply:
Yes it is just duplicated line (missed or some merge result).